.github/CODEOWNERS

```
@@ -1,3 +1,9 @@
-** @acoates-ms
-** @tom-un
-** @alloy
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# the global will be requested for
+# review when someone opens a pull request.
+* @HeyImChris @acoates-ms @Saadnajmi @amgleitman
```

@harrieshin Do we have / can we create an alias for React Native macOS like the [FluentUI Native](https://github.com/microsoft/fluentui-apple/blob/7674a2385af8fd4c352316f535fc90050d4c6ddb/.github/CODEOWNERS#L6) one?

---

yes apple-rn-team would be the better use here

---

is @apple-rn-team a valid GH alias? 🤔 or is it like @microsoft/apple-rn-team?

---

that said I don't think it'd be a great idea to get all that team (12 people as far as I can see?) getting pinged on every PR that gets opened against this repo? I'm fine if you are ok with it, just wanted to make sure to highlight that it'd probably generate a bit of noise for some folks?

---

yes, we use the microsoft GitHub Team alias a lot so that it is easier to manage/transfer permission and also helps when people go on vacation and such other teammates can help fill in the gap. We do that for other repository as well. for your "spam" concern, usually people filter their own inbox or notification.

---

ok so this is preferred, right?
```suggestion
* @microsoft/apple-rn-team
```